zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -5862,33 +6184,65 @@ components:
           $ref: "#/components/schemas/SortOrderEnum"
       required:
         - field
-    AuthorizationSearchQueryRequest:
+    AuthorizationSearchQueryRequestBase:
+      description: Base properties for AuthorizationSearchQueryRequest
+      type: object
       allOf:
         - $ref: "#/components/schemas/SearchQueryRequest"
-      type: object
       properties:
         sort:
           description: Sort field criteria.
           type: array
           items:
             $ref: "#/components/schemas/AuthorizationSearchQuerySortRequest"
+    AuthorizationSearchQueryRequest:
+      deprecated: true
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/AuthorizationSearchQueryRequestBase"
+      properties:
         filter:
           description: The authorization search filters.
           allOf:
             - $ref: "#/components/schemas/AuthorizationFilterRequest"
-    AuthorizationFilterRequest:
-      description: Authorization search filter.
+    AuthorizationSearchQuery:
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/AuthorizationSearchQueryRequestBase"
+      properties:
+        filter:
+          description: The authorization search filters.
+          allOf:
+            - $ref: "#/components/schemas/AuthorizationFilter"
+    AuthorizationFilterRequestBase:
+      description: Base properties for AuthorizationFilterRequest
       type: object
       properties:
         ownerType:
           description: The type of the owner of permissions.
           type: object
           allOf:
             - $ref: "#/components/schemas/OwnerTypeEnum"
+    AuthorizationFilterRequest:
+      description: Authorization search filter.
+      deprecated: true
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/AuthorizationFilterRequestBase"
+      properties:
         ownerKey:
           description: The id of the owner of permissions.
           type: integer
           format: int64
+    AuthorizationFilter:
+      description: Authorization search filter. Key attributes as string values.
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/AuthorizationFilterRequestBase"
+      properties:
+        ownerKey:
+          description: The id of the owner of permissions.
```

```suggestion
          description: The ID of the owner of permissions.
```

Is "the owner of permissions" the admin?

---

It's just the unique key (or ID) of the user, if I'm not mistaken.